tex/gloss-korean.ldf

```diff
@@ -0,0 +1,192 @@
+\ProvidesFile{gloss-korean.ldf}[polyglossia: module for Korean]
+
+\PolyglossiaSetup{korean}{
+    script=Hangul,
+    scripttag=hang,
+    language=Korean,
+    langtag=KOR,
+    hyphennames={english,american,usenglish,USenglish},
```

Comment:
It is intentional. 

Korean language comprise Hangul, Hanja (CJK Ideographs), ASCII, Kana (Japanese), Greek, Cyrillic, and some symblic characters.  In Korea, ASCII letters are frequently used side by side with Hangul, especially in academic articles. If we disable hyphenation of ASCII letters, It would give us much inconveniency as we have to switch language every time we want to write just one or two english words or phrases.  

If this setting affects typesetting of other languages or decreases computer performance significantly, then `nohyphenation` must be unavoidable. Otherwise, keeping the current setting seems to be better. (if possible, enabling British english as well would be even better.)